notebooks/llm-agent/llm-agent.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>please add link on provided optimizations in docs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1894/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
done

Reply:
I mean this doc https://docs.openvino.ai/2024/learn-openvino/llm_inference_guide/llm-inference-hf.html#enabling-openvino-runtime-optimizations